financial-connections/src/main/java/com/stripe/android/financialconnections/features/networkingsavetolinkverification/NetworkingSaveToLinkVerificationViewModel.kt

```diff
@@ -50,13 +52,16 @@ internal class NetworkingSaveToLinkVerificationViewModel @Inject constructor(
         logErrors()
         suspend {
             val consumerSession = requireNotNull(getCachedConsumerSession())
+            // If we automatically moved to this pane due to prefilled email, we should show the "Not now" button.
+            val showNotNowButton = getManifest().accountholderCustomerEmailAddress != null
```

Comment:
Does this still work if the user goes back and updates their email address to a different one, then re-enters this screen? Is the manifest updated to reflect that?